data/net/minecraft/world/entity/ai/goal/WrappedGoal.mapping

```diff
@@ -1,17 +1,22 @@
 CLASS net/minecraft/world/entity/ai/goal/WrappedGoal
+	METHOD <init> (ILnet/minecraft/world/entity/ai/goal/Goal;)V
+		COMMENT This is an internal object used by the GoalSelector to choose between Goals.
+		COMMENT In most cases, it should not be constructed directly.
+		COMMENT
+		COMMENT For information on how individual methods work, see the javadocs for Goal:
+		COMMENT {@link net.minecraft.entity.ai.goal.Goal}
```

Comment:
This comment should be on the class.
Additionally:
`For information on how individual methods work, see  {@link net.minecraft.entity.ai.goal.Goal}.`